### ISSUE TITLE
Steam Userdata Folder Save Location

### ISSUE THREAD

**Issue by ugurkahriman** [otavepto/gbe_fork#62]
Greetings! Can you make the Steam userdata folder save to the Goldberg SteamEmu Saves folder? So like other Steam Emu's (CODEX). This way we can use the Denuvo token without changing the Steam folder.

**Comment by Detanup01** [comment]
If you find the code where it want to look for steam userdata folder I happily change it but I didnt see anywhere

**Comment by Detanup01** [comment]
yeah I made along the way and still didnt see any mention to Steam UserData folder, might be some game calling getfile with system stuff and not from steam emu, which other PR #50 talking about kinda.

**Comment by Noisfex** [comment]
For example, Football Manager 2024 is protected by Denuvo and requires a Denuvo token for the MKDEV crack (like other Denuvo games). The Denuvo token is normally located in the Steam/userdata folder for cracked/uncracked games. In CODEX Emu we can redirect it somewhere else. I wonder if we can do the same for Goldberg Emu?

![resim](https://github.com/otavepto/gbe_fork/assets/160227503/dd1b4b0f-15d4-4816-bc79-3efed55fd960)

**Comment by Noisfex** [comment]
For example, Football Manager 2023 is protected by Denuvo and requires a Denuvo token for the MKDEV crack (like other Denuvo games). The Denuvo token is normally located in the Steam\userdata folder for cracked/uncracked games. In CODEX Emu we can redirect it somewhere else. I wonder if we can do the same for Goldberg Emu?

![305491843-dd1b4b0f-15d4-4816-bc79-3efed55fd960](https://github.com/otavepto/gbe_fork/assets/160227503/3ddec1b2-4be0-45c6-8ca5-e11b6abfa288)
